assets/src/edit-story/components/library/shapeLibrary.js

```diff
@@ -18,43 +18,92 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+
 /**
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { MASKS } from '../../masks';
+import { Section, Title, SearchInput, Header } from './common';
+
+const PREVIEW_SIZE = 36;
+
+const ShapePreview = styled.div`
+  position: relative;
+  display: inline-block;
```

Comment:
ugh. any other way?

Reply:
is `flex-wrap: wrap` better?

Reply:
IMHO, by far better :) I know that elsewhere we may have to use some special two-column layering component. It might be reusable here as well. But if all of our shapes will always be the same dimensions that may be unnecessary for now.